grafana/perf/regression-summary.json

```diff
@@ -70,20 +71,284 @@
+                },
+                "41021": {
+                  "index": 28,
+                  "text": "4.10.2-1"
+                },
+                "41023": {
+                  "index": 29,
+                  "text": "4.10.2-3"
+                },
+                "41054": {
+                  "index": 49,
+                  "text": "4.10.5-4"
+                },
+                "41066": {
+                  "index": 48,
+                  "text": "4.10.6-6"
+                },
+                "41110": {
+                  "index": 60,
+                  "text": "4.11.10"
+                },
+                "41111": {
+                  "index": 61,
+                  "text": "4.11.11"
+                },
+                "41112": {
+                  "index": 62,
+                  "text": "4.11.12"
+                },
+                "41113": {
+                  "index": 63,
+                  "text": "4.11.13"
+                },
+                "41114": {
+                  "index": 64,
+                  "text": "4.11.14"
+                },
+                "49211": {
+                  "index": 7,
+                  "text": "4.9.2-11"
+                },
+                "410129": {
+                  "index": 22,
+                  "text": "4.10.1-29"
+                },
+                "410136": {
+                  "index": 23,
+                  "text": "4.10.1-36"
+                },
+                "410160": {
+                  "index": 24,
+                  "text": "4.10.1-60"
+                },
+                "410170": {
+                  "index": 25,
+                  "text": "4.10.1-70"
+                },
+                "410197": {
+                  "index": 26,
+                  "text": "4.10.1-97"
+                },
+                "411115": {
+                  "index": 52,
+                  "text": "4.11.1-15"
+                },
+                "411121": {
+                  "index": 51,
+                  "text": "4.11.1-21"
+                },
+                "411135": {
+                  "index": 46,
+                  "text": "4.11.1-35"
+                },
+                "411142": {
+                  "index": 47,
+                  "text": "4.11.1-42"
+                },
+                "4100683": {
+                  "index": 17,
+                  "text": "4.10.0-683"
+                },
+                "4100688": {
+                  "index": 18,
+                  "text": "4.10.0-688"
+                },
+                "4100700": {
+                  "index": 19,
+                  "text": "4.10.0-700"
+                },
+                "4100729": {
+                  "index": 20,
+                  "text": "4.10.0-729"
+                },
+                "4101101": {
+                  "index": 27,
+                  "text": "4.10.1-101"
+                },
+                "4110137": {
+                  "index": 50,
+                  "text": "4.11.0.137"
```

Comment:
Should this be `4.11.0-137`?

Reply:
I prefer leave the version without special character, I dont see any reason to update it for every version

Reply:
My point is that all of the others are of the form "4.11.0-137" while this is "4.11.0.137".  They should be consistent -- either all periods or all hyphens.

Reply:
Ok I will fix it